Sources/TuistLoader/Models+ManifestMappers/Project+ManifestMapper.swift

```diff
@@ -31,13 +31,18 @@ extension XcodeGraph.Project {
         let developmentRegion = manifest.options.developmentRegion
         let options = XcodeGraph.Project.Options.from(manifest: manifest.options)
         let settings = try manifest.settings.map { try XcodeGraph.Settings.from(manifest: $0, generatorPaths: generatorPaths) }
+        let targetType = switch type {
+        case .local: TargetType.local
+        case .external: TargetType.remote
+        }
```

Comment:
```suggestion
        let targetType: TargetType = switch type {
        case .local: .local
        case .external: .remote
        }
```

This way, we can specify the type just once.